packages/grafbase-sdk/src/config.ts

```diff
@@ -41,9 +41,7 @@ export class GraphConfig {
   private readonly cache?: GlobalCache
   private readonly experimental?: Experimental
 
-  constructor(input: GraphConfigInput)
   /** @deprecated use `graph` instead of `schema` */
-  constructor(input: DeprecatedGraphConfigInput)
```

Comment:
These two overloads were causing me type errors in jest for some weird reason, so got rid of them.

They seem equivalent to the constructor below anyway...